recovery/src/main/java/com/kin/ecosystem/recovery/backup/presenter/BackupInfoPresenterImpl.java

```diff
@@ -20,6 +20,7 @@ public BackupInfoPresenterImpl(@NonNull CallbackManager callbackManager,
 		this.backupNavigator = backupNavigator;
 		this.callbackManager = callbackManager;
 		this.callbackManager.sendBackupEvent(BACKUP_WELCOME_PAGE_VIEWED);
+		this.callbackManager.setCancelledResult(); // make sure cancel will be called if nothing happened.
```

Comment:
mmm , when does it happen? what does it mean when nothing has happened?

Reply:
enter the backup flow and press back immediately.

Reply:
don't we have some back/close handling?